app/assets/javascripts/dashboard/protocols.js.coffee

```diff
@@ -191,3 +201,11 @@ $(document).ready ->
           type: 'get'
           url: "/dashboard/protocols.js"
           data: data
+      # Protocol Table Sorting End #
+
+
+
+(exports ? this).reset_service_requests_handlers = ->
```

Comment:
Condensed this code segment to a global function instead of using it all over the place